openmdao/core/system.py

```diff
@@ -1097,9 +1174,9 @@ def _setup_bounds(self, root_lower, root_upper, resize=False):
         """
         vector_class = root_lower.__class__
         self._lower_bounds = lower = vector_class(
-            'lower', 'output', self, root_lower, resize=resize)
+            'nonlinear', 'output', self, root_lower, resize=resize)
```

Comment:
I changed these to 'nonlinear' because there are various data structures that are keyed on vector name that the vector uses internally, like _var_sizes for example.  The 'nonlinear' and 'linear' data structures are the only ones that don't have reduced contents due to relevance reduction, so I could have used either 'linear' or 'nonlinear' here.

Reply:
Ah, I see